src/lib/Preview.js

```diff
@@ -702,6 +702,10 @@ class Preview extends EventEmitter {
         // Optional additional query params to append to requests
         this.options.queryParams = options.queryParams || {};
 
+        // Option to pause requireJS while Preview loads third party dependencies
+        // RequireJS will be re-enabled on the 'assetsloaded' event fired by Preview
+        this.options.pauseRequireJS = !!options.pauseRequireJS;
```

Comment:
still not 100% on this name.

Reply:
`disableRequireJS`, since you've already mentioned in the comment that it `will be re-enabled on the 'assetloaded' event`?

Reply:
Another argument for `disableRequireJS`, you use that term later on as parameter names

Reply:
I feel like disable means that it won't come back

Reply:
disableRequireJSWhileLoadingAssets :)